tick/inference/base/learner_optim.py

```diff
@@ -172,6 +185,12 @@ def __init__(self, penalty='l2', C=1e3, solver="svrg", step=None,
                         penalty == 'elasticnet':
             self.elastic_net_ratio = elastic_net_ratio
 
+        if 'blocks_start' in self._actual_kwargs and penalty == 'binarsity':
```

Comment:
should be a `or` not a `and`.
The idea is that if user set parameter `blocks_start` with a l2 penalty it should get a warning